### DISCUSSION TITLE
The language 'clang' is not loaded, falling back to 'txt' for syntax highlighting.

### DISCUSSION THREAD

**Discussion by elarbor** [vuejs/vitepress#3727] [Q&A]
项目配置搜索之后就报错：
![image](https://github.com/vuejs/vitepress/assets/43592730/9e3d60ea-f46a-4ba8-9161-ee3ca7f7b050)

**Comment by brc-dd** [comment]
The clang error is because you've something like this in one of your markdown files:

````
```clang
````

`clang` is not a valid language. You probably need `c` or `cpp` there instead of `clang`.

The ENOENT error is because you've some left over code from merge conflict. Search for `<<<<<<<` in your code and remove it properly.
